src/bundle/Resources/views/themes/admin/ui/page_title_edit_content.html.twig

```diff
@@ -0,0 +1,19 @@
+{% block close_button %}
```

Comment:
If this file is used in content edit iI think it should in the `content` catalog